flow-preferences/src/main/java/com/fredporciuncula/flow/preferences/FlowSharedPreferences.kt

```diff
@@ -4,70 +4,154 @@ import android.content.SharedPreferences
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.flow.Flow
 import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
 
 internal typealias KeyFlow = Flow<String?>
 
+/**
+ * Specific wrapper, that presents wide range of methods for handful using of [SharedPreferences] via [Preference] wrapper
+ * @param sharedPreferences takes an instance of [SharedPreferences]
+ * @param coroutineContext takes a [CoroutineContext]. Can be used to override standard behavior, e.g. logging
+ */
 class FlowSharedPreferences @JvmOverloads constructor(
   val sharedPreferences: SharedPreferences,
-  val coroutineContext: CoroutineContext = Dispatchers.IO
+  val coroutineContext: CoroutineContext = EmptyCoroutineContext,
 ) {
 
   private val keyFlow: KeyFlow = sharedPreferences.keyFlow
 
+  /**
+   * Creates a [Preference] wrapper for [Int] by a given key from [SharedPreferences]. By default, all operations will be executed on [Dispatchers.IO]
+   * @param key key for a [SharedPreferences] storage value
+   * @param defaultValue will be returned if value by given key is not found
+   * @return [Preference] wrapper of type [Int]
+   */
   @JvmOverloads
   fun getInt(key: String, defaultValue: Int = 0): Preference<Int> =
```

Comment:
💡 To keep it short, I think we could try mentioning everything in a single sentence. The arguments are pretty self-explanatory, and we already have more context on them in the documentation of the `Preference` itself, so I think we can be pretty succinct here:

>Creates a [Preference] of type [Int] with the given `key` and `defaultValue`.